### PR TITLE
Yel 2673

### PR DESCRIPTION
**Pull request by JWanderema**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -183,6 +183,186 @@ EOF
   }
 }
 
+resource "aws_instance" "ext_tableau_linux_staging" {
+  count                       = "${var.environment == "prod" ? "2" : "1"}" # Allow different instance count in prod and notprod
```
Comment (ianrelf) [inline main.tf]: count                       = "${var.environment == "prod" ? "1" : "0"}"

File: rds.tf [hunk 2]
```diff
@@ -1,3 +1,7 @@
+locals {
+  external_reporting_stg_count = "${var.environment == "prod" ? "1" : "1"}"
```
Comment (ianrelf) [inline rds.tf]: external_reporting_stg_count = "${var.environment == "prod" ? "1" : "0"}"

File: variables.tf [hunk 3]
```diff
@@ -145,6 +145,19 @@ variable "dq_external_dashboard_instance_ip" {
   ]
 }
 
+variable "dq_external_staging_dashboard_instance_ip" {
+  description = "IP addresses of EC2 instances in Prod"
+  type        = "list"
+
+  default = [
+    "10.1.14.115",
```
Comment (ianrelf) [inline variables.tf]: Like with Tab Int Stg, there will only be one Tab Ext Stg.

File: rds.tf (hunk 2 above)
Comment (david-wirelab) [reply rds.tf]: As Ian suggested this still need changing.

File: main.tf [hunk 4]
```diff
@@ -183,6 +183,186 @@ EOF
   }
 }
 
+resource "aws_instance" "ext_tableau_linux_staging" {
+  count                       = "${var.environment == "prod" ? "1" : "0"}" # Allow different instance count in prod and notprod
+  key_name                    = "${var.key_name}"
+  ami                         = "${data.aws_ami.ext_tableau_linux.id}"
+  instance_type               = "${var.environment == "prod" ? "c5.2xlarge" : "c5.2xlarge"}"
```
Comment (david-wirelab) [inline main.tf]: Given there won't be an instance in NotProd this line should be changed.

**Pull request merged**